BatteryManagementConfig.h

```diff
@@ -0,0 +1,93 @@
+typedef struct
+{
+	float Low_Critical_range;	// Lower Limit to alert critical error
+	float Low_Warning_range;	// Lower Limit to warn
+	float High_Warning_range; 	// Higher Limit to alert critical error
+	float High_Critical_range;	// Higher Limit to warn
+}BatteryLimitConfig;
+
+
+typedef struct
+{  // Index 0 - Alert in English 1 - Alert in German
+	char Low_Critical_Alertmessage [2][50];	// Alert message when measured value is lower than Low_Critical_range
+	char Low_Warning_Alertmessage [2][50];	// Warning message when measured value is < Low_Warning_range and > Low_Critical_range value
+	char Normal_Alertmessage [2][50];	// Alert message when measured value is in normal_range value
+	char High_Warning_Alertmessage [2][50];	// Warning message when measured value is > High_Warning_range and < High_Critical_range value
+	char High_Critical_Alertmessage [2][50];// Alert message when measured value is Higher than High_Critical_range
+}BatteryAlertmessageConfig;
+
+
+
+typedef struct
+{    
+	BatteryLimitConfig Limit;
+	BatteryAlertmessageConfig Alertmessage;
+}BatteryManagementConfig;
+ 
+
+
+BatteryManagementConfig temperature_config=	\
+{\

```

Comment:
Excellent separation of data from logic.
Suggest to separate data with different reasons for change. A Language gets added when we sell our product in a new region. Limits can change for other reasons.